end-end-tests/api-standards/test-bulk.bash

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 set -eu
 HERE=$(cd $(dirname $0); pwd)
+export CI=""
```

Comment:
Is this a new requirement for `compare` / `bulk-compare`? Seems like an unexpected change in behavior.

Reply:
Ah, missed @acunniffe 's comment above -- we're likely hitting this _because_ we're running this in Circle, but "offline". I'll test this locally.

Reply:
>  Is this a new requirement for compare / bulk-compare? Seems like an unexpected change in behavior.

It should not impact running locally, unless there is a local `CI=true` environment variable set. 

It should not affect the CI use case today. 

However, we did fix the regression and have a new version coming out that should have the same behaviors as before. If we wait 20-30 mins for that to release, I can update this to remove the need for these changes to the tests